src/components/search_bar/query/date_format.ts

```diff
@@ -149,7 +149,7 @@ const parseWeek = (value: string) => {
       parsed = utc().add(1, 'weeks');
       break;
     default:
-      const match = value.match(/week ([1-9][1-9]?)/i);
+      const match = value.match(/week ([0-9][0-9]?)/i);
```

Comment:
```suggestion
      const match = value.match(/week (\d+)/i);
```

`[0-9]` is identical to `\d`, and `+` expresses the one-or-more expectation